tests/unit/rpc/handlers/AccountInfoTests.cpp

```diff
@@ -655,6 +655,49 @@ TEST_F(RPCAccountInfoHandlerTest, IdentAndSignerListsFalse)
     });
 }
 
+TEST_F(RPCAccountInfoHandlerTest, EmptySignerLists)
+{
+    auto const ledgerHeader = createLedgerHeader(kLEDGER_HASH, 30);
+    EXPECT_CALL(*backend_, fetchLedgerBySequence).Times(1);
+    ON_CALL(*backend_, fetchLedgerBySequence).WillByDefault(Return(ledgerHeader));
```

Comment:
I know that this is probably copied from another test, but let's use one `EXPECT_CALL().WillOnce()` where possible instead of `EXPECT_CALL()` and `ON_CALL()`